public/app.css

```diff
@@ -0,0 +1,29 @@
+.visually-hidden {
+  position: absolute !important;
+  height: 1px;
+  width: 1px;
+  overflow: hidden;
+  clip: rect(1px 1px 1px 1px); /* IE6, IE7 */
```

Comment:
do we even support IE6, IE7?

Reply:
In the [browserlist](https://github.com/excalidraw/excalidraw/blob/d171e9705d961c272b8a001d2ead62a8ffe1d5ab/package.json#L6) its mentioned we target IE11 onwards.
 I think this might have gotten missed when adding the `visually-hidden` class. 
lets remove it ?

Reply:
yes, remove it.